go/src/lib.rs

```diff
@@ -379,6 +394,70 @@ pub unsafe extern "C" fn command(
                 command_response.float_value = num;
                 Ok(Some(command_response))
             }
+            Value::Boolean(bool) => {
+                command_response.bool_value = bool;
+                Ok(Some(command_response))
+            }
+            Value::Array(array) => {
+                let len = array.len();
+                let mut vec_elements_len: Vec<c_long> = Vec::<c_long>::with_capacity(len);
+                let vec = array
+                    .iter()
```

Comment:
Why `iter` and not `into_iter`? I think `array` isn't being used anywhere else, so it can be consumed here, similar to Maps.